compliance/virtualmachines/agent/index/index.go

```diff
@@ -0,0 +1,98 @@
+package index
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/stackrox/rox/compliance/node/index"
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/pkg/httputil/proxy"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/compliance/virtualmachines/agent/config"
+	"github.com/stackrox/rox/compliance/virtualmachines/agent/vsock"
+)
+
+var log = logging.LoggerForModule()
+
+const (
+	mappingClientTimeout = 30 * time.Second
+	repo2CPEMappingURL   = "https://security.access.redhat.com/data/metrics/repository-to-cpe.json"
+)
```

Comment:
In my demo code, I took a shortcut calling this Internet URL, but this data should be exposed also by Sensor using internal http API. You may consider making this configurable through env or flag so that the users may choose the location of this file.